package/knot-control/knot-control.mk

```diff
@@ -7,6 +7,7 @@
 KNOT_CONTROL_VERSION = KNOT-v02.00
 KNOT_CONTROL_SITE = https://github.com/CESARBR/knot-gateway-control.git
 KNOT_CONTROL_SITE_METHOD = git
+KNOT_CONTROL_DEPENDENCIES = python-pika python-pymongo
```

Comment:
The knot-control has other dependencies such as:
 - BR2_PACKAGE_PYTHON_SETUPTOOLS
 - BR2_PACKAGE_PYTHON_LOCKFILE
 - BR2_PACKAGE_PYTHON_GOBJECT
 - BR2_PACKAGE_DBUS_PYTHON
 - BR2_PACKAGE_PYTHON_DOCUTILS
 - BR2_PACKAGE_PYTHON_DAEMON

I think these dependencies should be on this file and on the knot-control Config.in file.